go.sum

```diff
@@ -0,0 +1,16 @@
+github.com/davecgh/go-spew v1.1.0/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
```

Comment:
testify is great, but one goal I've been trying to adhere to is to not pull in any other dependencies unless they are really necessary. Can you use the standard library functionality for this?

Reply:
Ok, done

Reply:
Awesome, thanks!